rules-tests/CodeQuality/Rector/Catch_/ThrowWithPreviousExceptionRector/Fixture/skip_filled_exception_with_different_location.php.inc

```diff
@@ -18,7 +18,7 @@ class SkipFilledExceptionWithDifferentLocation
 
 class BadRequestHttpException extends  Exception
 {
-    public function __construct(string $message = null, \Throwable $previous = null, int $code = 0, array $headers = [])
+    public function __construct(?string $message = null, ?\Throwable $previous = null, int $code = 0, array $headers = [])
```

Comment:
PHP Deprecated: Implicitly marking parameter as nullable is deprecated, the explicit nullable type must be used instead

Reply:
please revert fixture files with `.php.inc` suffix, test fixtures are not shipped on release